src/main/resources/messages.properties

```
@@ -1,3 +1,70 @@
 login.id=id
 login.password=password
-login.signIn=Sign in
\ No newline at end of file
+login.signIn=Sign in
+
+add=\uCD94\uAC00
```

인텔리제이 설정을 적절히 변경할 것(설정->파일 인코딩->프로퍼티 파일에 대한 디폴트 인코딩을 UTF-8로 설정하고, '명확한 Native에서 ASCII로의 변환'을 체크하라)
```suggestion
add=\uCD94\uAC00
```

---

수정하겠네